mods/ra/weapons/other.yaml

```diff
@@ -177,6 +208,22 @@ TTankZap:
 			None: 1000
 		DamageTypes: Prone50Percent, TriggerProne, ElectricityDeath
 
+TTankCharge:
+	ReloadDelay: 120
+	Range: 2c0
+	Report: TESLACHARGE.AUD
+	Charges: yes
+	ValidStances: Ally
+	ValidTargets: TeslaBoost
+	Projectile: TeslaZap
+		Image: litningupg
+	Warhead@charge: GrantUpgrade
+		Range: 42
+		Duration: 125
+		Upgrades: tesla, tesla1, tesla2
```

Comment:
How difficult would it be to add an `UpgradeLevels:` parameter that we set to 3 for the tesla tank?